src/components/ReportActionItem/RenameAction.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import Text from '../Text';
+import styles from '../../styles/styles';
+import reportActionPropTypes from '../../pages/home/report/reportActionPropTypes';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+
+const propTypes = {
+    /** All the data of the action */
+    action: PropTypes.shape(reportActionPropTypes).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+const RenameAction = (props) => {
+    const displayName = lodashGet(props.action, ['message', 0, 'text']);
+    const oldName = lodashGet(props.action, 'originalMessage.oldName', '');
+    const newName = lodashGet(props.action, 'originalMessage.newName', '');
+
+    return (
+        <Text style={[styles.pv2, styles.ph5, styles.textAlignCenter, styles.textLabelSupporting]}>
```

Comment:
could we try this with `.pv3` and see how it looks? That's what we had in our mockups so just curious:

![image](https://user-images.githubusercontent.com/2319350/150306292-cb13944f-3375-4287-a71a-2c971d3ae181.png)

Reply:
Here it is with pv3. I think I like pv3 better actually.

**Web**
![web-pv3](https://user-images.githubusercontent.com/22219519/150397925-1ef36361-5c04-463d-ba79-e0929cecf199.png)

**iOS**
![ios-pv3](https://user-images.githubusercontent.com/22219519/150397954-d53ff039-d494-488d-9f9c-f3e75619a777.png)

**Android**
![android-pv3](https://user-images.githubusercontent.com/22219519/150397976-caf1e741-eb32-4255-8dff-7f70bea18fef.png)

Reply:
Agree, let's do .pv3. Thanks!